test/unit-test/ota_os_posix_utest.c

```diff
@@ -172,21 +171,21 @@ void test_OTA_posix_InvalidTimerOperations( void )
 {
     OtaErr_t result = OTA_ERR_UNINITIALIZED;
 
-    result = timer.start( timer.PTimerCtx, TIMER_NAME, OTA_DEFAULT_TIMEOUT, NULL );
+    result = timer.start( timer_id, TIMER_NAME, OTA_DEFAULT_TIMEOUT, NULL );
     TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
 
     /* Set the timeout to 0 and stop the timer*/
-    result = timer.start( timer.PTimerCtx, TIMER_NAME, 0, NULL );
+    result = timer.start( timer_id, TIMER_NAME, 0, NULL );
     TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
 
-    result = timer.stop( timer.PTimerCtx );
-    TEST_ASSERT_NOT_EQUAL( OTA_ERR_NONE, result );
+    result = timer.stop( timer_id );
+    TEST_ASSERT_EQUAL( OTA_ERR_NONE, result );
```

Comment:
Can we move this before we start a timer and expect to fail the timer stop?   TEST_ASSERT_NOT_EQUAL( OTA_ERR_NONE, result );

Reply:
This pr is to fix the tests first, we can add more tests for failing path later.